frontend/src/components/Conversation/Conversation.tsx

```diff
@@ -74,15 +74,24 @@ export const Conversation = () => {
     (conn) => conn.id === currConversation?.connection_id
   );
 
-  const scrollToBottom = () => {
+  const scrollToBottom = (behavior: ScrollBehavior = "instant") => {
     if (messageListRef.current !== null) {
-      window.scrollTo({ top: messageListRef.current?.offsetTop });
+      window.scrollTo({ top: messageListRef.current?.offsetTop, behavior });
     }
   };
   useEffect(() => {
-    scrollToBottom();
+    // Wait for charts to render
+    const animationFrameId = requestAnimationFrame(() => scrollToBottom());
+
+    return () => cancelAnimationFrame(animationFrameId);
```

Comment:
What is this for? Maybe add some comments here to explain the behavior in detail